### ISSUE TITLE
LAD and TA names overlap

### ISSUE THREAD

**Issue by dabreegster** [acteng/atip#19]
Blackpool is in the geojson as both a LAD and TA, but differently. One is silently unselectable. We could scope all the names with LAD/ and TA/ or something if this is a problem

**Comment by Robinlovelace** [comment]
Yeah I think there will be more examples. The TA name should take precedence. Seeing as we're only using the double-boundary file https://github.com/acteng/schema/blob/main/boundaries/lads_tas_combined.geojson for this purpose, and the code that created it is in that repo, I suggest migrating this issue to there. Will give it a go.
